cmd/config/internal/commands/cmdset.go

```diff
@@ -32,18 +34,48 @@ func NewSetRunner(parent string) *SetRunner {
 		"annotate the field with who set it")
 	c.Flags().StringVar(&r.Perform.Description, "description", "",
 		"annotate the field with a description of its value")
+	c.Flags().StringVar(&setterVersion, "version", "",
+		"use this version of the setter format")
+	c.Flags().MarkHidden("version")
 
 	return r
 }
 
+var setterVersion string
+
+var GetOpenAPIFile = func(args []string) (string, error) {
+	return filepath.Join(args[0], "kustomization"), nil
```

Comment:
Is this kustomization or Kptfile?

Reply:
kustomization

Reply:
But the kpt binary is looking for file named kustomization and failing. We have decided to put it on the Kptfile correct? Are we now storing it in the file name kustomization ?